script.js

```diff
@@ -37,7 +37,7 @@ const callAnalytics = (email) => {
 const getRDSEmail = async () => {
 	const email = await axios
 		.get(
-			"https://api.realdevsquad.com/users/self?private=true",
+			"https://api.realdevsquad.com/users?profile=true",
```

Comment:
I read the website-api-contracts since i have to make changes there too. In it was mentioned that when the /self?private=true is used it will also send the phone number and email in the object that is being sent back. and /users?profile=true is a alternative for the deprecated api endpoint /users/self

## These are the outputs for each one of those:
### /self?private=true
![sels private=true](https://github.com/user-attachments/assets/a02a4505-854a-41d1-bbfa-7d6eaa79dbb5)

### /users?profile=true
![user profile=true](https://github.com/user-attachments/assets/1c0e5954-2d2a-4d53-aaad-de45e52f1b50)

The output is same in both cases

Reply:
Are the details like email or phone being used in the project?

Reply:
Phone i am sure is not being used as far as i have inspected the code. But the email is input by the user on frontend, so it might be used to verify if the user is on RDS or not.